tox.ini

```diff
@@ -29,6 +29,7 @@ commands = sphinx-build docs docs/_build/html -W -b html
 
 # This env requires tox-conda.
 [testenv:figure]
+changedir = {toxinidir}
```

Comment:
@nabobalis I put this here so that when you run the tests locally you get access to the generated files. We don't need the C extensions for this anyway right?

Reply:
That should be ok but you will need to update the docs, and the circleci path to match.

Reply:
When I ran it locally, it just made a tmp dir in the sunpy repo folder, it didn't actually use my real tmp volumne.